felix/fv/rpf_test.go

```diff
@@ -150,12 +150,48 @@ var _ = infrastructure.DatastoreDescribe(
 			log.Info("AfterEach done")
 		})
 
+		Context("With BPFEnforceRPF=Disabled", func() {
+			BeforeEach(func() {
+				options.ExtraEnvVars["FELIX_BPFEnforceRPF"] = "Disabled"
+			})
+
+			It("should allow packets from wrong direction with disabled RPF on main device", func() {
+				fakeWorkloadIP := "10.65.15.15"
+
+				tcpdumpHEP := felixes[0].AttachTCPDump("eth20")
+				tcpdumpHEP.SetLogEnabled(true)
+				matcherHEP := fmt.Sprintf("IP %s\\.30446 > %s\\.30446: UDP", fakeWorkloadIP, w.IP)
+				tcpdumpHEP.AddMatcher("UDP-30446", regexp.MustCompile(matcherHEP))
+				tcpdumpHEP.Start()
+				defer tcpdumpHEP.Stop()
+
+				tcpdumpWl := w.AttachTCPDump()
+				tcpdumpWl.SetLogEnabled(true)
+				matcherWl := fmt.Sprintf("IP %s\\.30446 > %s\\.30446: UDP", fakeWorkloadIP, w.IP)
+				tcpdumpWl.AddMatcher("UDP-30446", regexp.MustCompile(matcherWl))
+				tcpdumpWl.Start()
+				defer tcpdumpWl.Stop()
+
+				_, err := external.RunCmd("pktgen", fakeWorkloadIP, w.IP, "udp",
+					"--port-src", "30446", "--port-dst", "30446", "--ip-id", "666")
```

Comment:
nit: a different id helps to distinguish and find the right packet in the logs

```suggestion
					"--port-src", "30446", "--port-dst", "30446", "--ip-id", "667")
```

Reply:
Updated as suggested